_includes/markdown/Design.md

```diff
@@ -171,12 +171,18 @@ If a management UI is impossible due to the nature of the project, credentials s
 <?php
 // Production API keys should ideally be defined in wp-config.php
 // This section should default to a development or noop key instead.
-if ( ! defined( 'CLIENT_MANDRILL_API_KEY' ) && ! ENV_DEVELOPMENT ) {
-	define( 'CLIENT_MANDRILL_API_KEY', '1234567890' );
+if ( ! defined( 'CLIENT_MANDRILL_API_KEY' ) {
+    if ( defined( 'WP_ENVIRONMENT_TYPE' ) && 'production' === wp_get_environment_type() ) {
```

Comment:
@jamesmorrison - This is picky, but you don't need `defined( 'WP_ENVIRONMENT_TYPE'  ) && `.

`wp_get_environment_type` already includes this check and if the value is not set it returns `''`

Reply:
@gthayer I removed the `defined( 'WP_ENVIRONMENT_TYPE' )` conditional.